### PR TITLE
super_distance

### PR DESCRIPTION
**Pull request by leomrtns**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).

The tests passed using "bootstrap.py"/"bioconda-utils build" and also with conda-build.
When running "circleci build" it complained about missing git.

### CODE REVIEW

**Comment by leomrtns** [comment]
@bioconda/core this is my first contribution, @dpryan79 helped me with a previous PR that wasn't successful. This is a program for supertrees estimation. 
cheers,
Leo

File: recipes/super_distance/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - automake
+    - make
+    - check
+    - libcxx 4.0.* # [osx]
+    - openmp 4.0.* # [osx]
+    - llvm-openmp # [osx]
```
Comment (dpryan79) [inline recipes/super_distance/meta.yaml]: Can you remove the 3 OSX lines? Since OSX is being skipped they're not needed.

Comment (dpryan79) [reply recipes/super_distance/meta.yaml]: You can remove them from both the build and host sections :)

Comment (leomrtns) [reply recipes/super_distance/meta.yaml]: done! tks for checking

**Comment by dpryan79** [comment]
Thanks @leomrtns!

**Pull request merged**